sgl-kernel/tests/test_moe_fused_gate.py

```diff
@@ -43,8 +47,30 @@ def test_moe_fused_gate_combined(seq_length, dtype, params):
         num_expert_group=num_expert_group,
         topk_group=topk_group,
         compiled=False,
+        n_share_experts_fusion=n_share_experts_fusion,
     )
 
+    # When n_share_experts_fusion > 0, ignore the comparison of the last topk dimension
+    if n_share_experts_fusion > 0:
+        original_indices = indices.clone()
+        original_ref_indices = ref_indices.clone()
+
+        indices = indices[:, :-1]
+        ref_indices = ref_indices[:, :-1]
+
+        valid_min = num_experts
+        valid_max = num_experts + n_share_experts_fusion
+        shared_indices = original_indices[:, -1]
+        shared_ref_indices = original_ref_indices[:, -1]
+        if shared_indices is not None:
```

Comment:
nit: wondering when it will be none, since it looks like `original_indices[:, -1]` which is a tensor